src/stdlib/list.prelude.ts

```diff
@@ -139,13 +139,6 @@ function enum_list(start, end) {
   return start > end ? null : pair(start, enum_list(start + 1, end));
 }
 
-// Returns the item in xs (assumed to be a list) at index n,
-// assumed to be a non-negative integer.
-// Note: the first item is at position 0
-
-function list_ref(xs, n) {
-  return n === 0 ? head(xs) : list_ref(tail(xs), n - 1);
-}
```

Comment:
Why do you need to make this builtin?
If this is for performance, I am okay.
But if this is because the original `list_ref` does not work, then it means there are some underlying problems.

Reply:
Just for performance, it works for normal list but not for infinite list (like f(x)=> pair(x,f(x+1))) because isn't thunk aware (since it's in the prelude). So to make it more powerful I put it in the built-in.

Reply:
ok!